pkgs/development/python-modules/torpy/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, cryptography
+, pytestCheckHook
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "torpy";
+  version = "1.1.6";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "torpyorg";
+    repo = "torpy";
+    rev = "v${version}";
+    sha256 = "sha256-Ni7GcpkxzAMtP4wBOFsi4KnxK+nC0XCZR/2Z/eS/C+w=";
+  };
+
+  propagatedBuildInputs = [
+    cryptography
+    requests
+   ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  # touches network
+  pytestFlagsArray = [ "tests/" "--ignore=tests/integration" ];
```

```suggestion

  disabledTestPaths =[
    # requires network
    "--ignore=tests/integration"
  ];
```
The tests directory should be automatically found.